client/components/account-card.jsx

```diff
@@ -265,7 +274,11 @@ export default class AccountCard extends React.Component {
     if (this.props.hideName) {
       return;
     }
-    return <h1 className='view-profile-name'>{this.props.username}</h1>;
+    if (this.state.route.path === 'view-other-accounts') {
+      return <h1 onClick={this.handleUsernameClick} className='view-profile-name other-profiles'>{this.props.username}</h1>;
+    } else {
+      return <h1 onClick={this.handleUsernameClick} className='view-profile-name'>{this.props.username}</h1>;
+    }
```

Comment:
Rather than add a click listener, we should put an anchor tag inside of the `h1`. It may need a styling update, but that's fine